devtools/klap-decrypt.py

```diff
@@ -0,0 +1,272 @@
+"""
+This code allow for the decryption of KlapV2 data from a pcap file.
+
+It will output the decrypted data to a file.
+This was designed and tested with a Tapo light strip setup using a cloud account.
+"""
+
+import codecs
+import json
+import re
+
+import pyshark
+from cryptography.hazmat.primitives import padding
+
+from kasa.credentials import Credentials
+from kasa.deviceconfig import (
+    DeviceConfig,
+    DeviceConnectionParameters,
+    DeviceEncryptionType,
+    DeviceFamily,
+)
+from kasa.klaptransport import KlapEncryptionSession, KlapTransportV2
+
+# These are the only variables that need to be changed in order to run this code.
+# ********** CHANGE THESE **********
+username = "hello@world.com"  # cloud account username (likely an email)
+password = "hunter2"  # cloud account password  # noqa: S105
+# the ip of the smart device as it appears in the pcap file
+device_ip = "192.168.1.100"
+pcap_file_path = "/path/to/my.pcap"  # the path to the pcap file
+output_json_name = (
+    "output.json"  # the name of the output file, relative to the current directory
+)
+# **********************************
```

Comment:
Please a main entrypoint as function (e.g., `cli()`, see https://github.com/python-kasa/python-kasa/blob/master/devtools/dump_devinfo.py#L211) to allow defining the options from command line.

Reply:
Done in 4ed43e113b0a488c845c90f8068946512719d4e6